src/main/java/sg/ncl/testbed_interface/Team2.java

```diff
@@ -75,6 +80,17 @@ public void setCreatedDate(String createdDate) {
         this.createdDate = createdDate;
     }
 
+    // reads the JSON representation of the ZonedDateTime, e.g. 1.4912345E
+    // converts to a proper ZonedDateTime object
+    // store as ZonedDateTime object because we want to sort by date using DataTables plugin
+    // using String will only sort alphabetically
+    public void setProcessedDate(String processedDate) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+        mapper.registerModule(new JavaTimeModule());
+        ZonedDateTime date = mapper.readValue(processedDate, ZonedDateTime.class);
+        this.processedDate = date;
+    };
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)